### PR TITLE
Readme readibility improvement

### PR DESCRIPTION
**Pull request by vrkansagara**

### CODE REVIEW

File: tests/test_DateTime.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ tests.test_DateTime = function (t) {
     localTZ = Math.abs(localTZ);
     localTZ = direction + ((localTZ < 10) ? "0" : "") + localTZ;
     testTimestamp = isoTimestamp("2005-2-3T22:01:03" + localTZ);
-    var testDateTimestamp = new Date(2005,1,3,22,1,3);
+    var testDateTimestamp = new Date(2005,1,3,22,31,3);
```
Comment (etrepum) [inline tests/test_DateTime.js:46]: I don't think this is correct, it fails here. If the existing test fails for you, what time zone is your computer set to?

```suggestion
    var testDateTimestamp = new Date(2005,1,3,22,1,3);
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# MochiKit Makes JavaScript Suck Less
```
Comment (etrepum) [inline README.md:1]: This looks good 👍

**Review by etrepum — CHANGES_REQUESTED** [review]

File: tests/test_DateTime.js (hunk 1 above)
Comment (vrkansagara) [reply tests/test_DateTime.js:46]: @etrepum  I am in IST timezone.

**Review by etrepum — APPROVED** [review]

File: tests/test_DateTime.js (hunk 1 above)
Comment (etrepum) [reply tests/test_DateTime.js]: Great catch! This looks like a longstanding issue with the test suite that has been there for over a decade. The implementation does not seem to have any issue with IST.

**Pull request merged**